tests/dashboard.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { test } from '@playwright/test'
 
 test('dashboard is on homepage', async ({ page }) => {
-  await page.goto('http://localhost:3000/')
+  await page.goto('http://127.0.0.1:3000')
```

Comment:
```suggestion
  await page.goto('/')
```

Reply:
I am not sure why though🤔. Just made another commit with your suggested changes.